### DISCUSSION TITLE
it shows only directories and it doesnt run a code

### DISCUSSION THREAD

**Discussion by grzegrzeNOW** [crewAIInc/crewAI#1148] [Q&A]
hi,

I am quite new in this environment. I have installed everything correctly:  version crewai==0.1.32 and I use python installed Python 3.12.1 After running a code it shows me two directories and thats it.

PS C:\Users\xxx\OneDrive\Documents\crewAI>  & C:/Users/xxx/AppData/Local/Programs/Python/Python312/python.exe c:/Users/xxx/AI.

after a while come back to: 

PS C:\Users\xxx\OneDrive\Documents\crewAI>
 
 How i should it solve this?
 Thanks for help

**Comment by Biancamazzi** [comment]
don't forget we have https://chat.openai.com/g/g-qqTuUWsBY-crewai-assistant to help and also the discord community https://discord.com/invite/X4JWnZnxPb

**Comment by Haripritamreddy** [comment]
Send the code here in which you are facing an error. You may have removed `print(result)` part of  the code so you may not see the result or smh.

**Comment by grzegrzeNOW** [comment]
yes, this is exactly what i did and it helped :)

pt., 9 lut 2024 o 17:19 haripritamreddy ***@***.***>
napisał(a):
